recipe/meta.yaml

```diff
@@ -25,10 +25,11 @@ requirements:
     - param >=1.6.1,<2.0
     - numpy >=1.0
     - matplotlib >=2.1
-    - bokeh >=0.12.15
+    - bokeh >=0.12.15,<=0.12.16
```

Comment:
Should this be added upstream as well? Also guessing this should be `<0.12.16` or are we ok with `0.12.16` and expect a break in `0.12.17`?

ref: https://github.com/ioam/holoviews/blob/v1.10.5/setup.py#L20

Reply:
We are actually compatible with 0.13.0 which is due to be released next week. It is pinned in the build uploaded to the pyviz channel which was built from this recipe: https://github.com/ioam/holoviews/blob/master/conda.recipe/meta.yaml#L28

Therefore we'll likely have another release shortly after the bokeh release to fix that.